### PR TITLE
Branch 22.06 bugs

### LINKED ISSUE #2173: [DOC] Number of vertices count inconsistent in Overlap Similarity Notebook
## Report incorrect documentation

**Location of incorrect documentation**
[notebook
](https://github.com/rapidsai/cugraph/blob/branch-22.04/notebooks/link_prediction/Overlap-Similarity.ipynb)
**Describe the problems or issues found in the documentation**

"The test graph has only 34 vertices, so why is the Graph listing 35?" But its listing 34.

Screenshot from 2022-03-29 13-46-39

**Suggested fix for documentation**
Remove note if this has been changed.

### PR DESCRIPTION
**Pull request by BradReesWork**
Fixing BUG

- closes #2173

### CODE REVIEW

**Review by ChuckHastings — APPROVED** [review]

**Comment by ChuckHastings** [comment]
rerun tests

File: python/cugraph/cugraph/tree/minimum_spanning_tree.py [hunk 1]
```diff
@@ -20,7 +20,7 @@
 
 def _minimum_spanning_tree_subgraph(G):
     mst_subgraph = Graph()
-    if type(G) is not Graph:
+    if G.is_directed():
         raise Exception("input graph must be undirected")
```
Comment (rlratzel) [inline python/cugraph/cugraph/tree/minimum_spanning_tree.py:24]:
```suggestion
        raise ValueError("input graph must be undirected")
```

File: python/cugraph/cugraph/tree/minimum_spanning_tree.py [hunk 2]
```diff
@@ -35,7 +35,7 @@ def _minimum_spanning_tree_subgraph(G):
 
 def _maximum_spanning_tree_subgraph(G):
     mst_subgraph = Graph()
-    if type(G) is not Graph:
+    if G.is_directed():
         raise Exception("input graph must be undirected")
```
Comment (rlratzel) [inline python/cugraph/cugraph/tree/minimum_spanning_tree.py:39]:
```suggestion
        raise ValueError("input graph must be undirected")
```

**Review by rlratzel — APPROVED** [review]

**Comment by rlratzel** [comment]
@gpucibot merge

**Pull request merged**